Dockerfile

```
@@ -31,7 +33,17 @@ RUN echo "@community http://dl-cdn.alpinelinux.org/alpine/edge/community" >> /et
     && rm -rf /tmp/* /var/tmp/* \
     && groupmod -g 1000 users \
     && useradd -u 911 -U -d /config -s /bin/false abc \
-    && usermod -G users abc
+    && usermod -G users abc \
+    && addgroup -S privoxy && adduser -S privoxy -G privoxy \
+    && mkdir /opt/privoxy \
+    && curl -sL https://www.privoxy.org/sf-download-mirror/Sources/3.0.29%20%28stable%29/privoxy-3.0.29-stable-src.tar.gz | tar -C /opt/privoxy --strip-components=2 -xz \
```

Hmm, I checked and it's a bit inconvenient that they don't have a "latest" tag or such to use for here..

---

Yep. The closest I could find to automatically becoming aware of Privoxy releases is to parse their release feed from here https://www.privoxy.org/feeds/privoxy-releases.xml 

Here is a one-liner to extract the latest source URL from the RSS feed and give it to `curl` to download from:

```suggestion
    && curl -sL $(curl -s https://www.privoxy.org/feeds/privoxy-releases.xml | egrep -o -m 1 'https:\/\/www\.privoxy\.org\/sf-download-mirror\/Sources\/[0-9]+\.[0-9]+\.[0-9]+%20\(stable\)\/privoxy-[0-9]+\.[0-9]+\.[0-9]+-stable-src\.tar\.gz<' | sed -n -e 's/<$//' -e 1p) | tar -C /opt/privoxy --strip-components=2 -xz \
```

And BTW - 3.0.29 is already 3 releases behind the current latest as of today: https://www.privoxy.org/sf-download-mirror/Sources/3.0.32%20%28stable%29/

---

3.0.29 is my cherry picked release for now as I found later 'stable' versions crashing after a few hours of use. :) Would love to use your one liner in future though!